src/setupTests.js

```diff
@@ -2,4 +2,8 @@
 // allows you to do things like:
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
-import '@testing-library/jest-dom/extend-expect';
+import "@testing-library/jest-dom/extend-expect";
+import { configure } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
```

Comment:
Remove this. only use @testing-library/react

Reply:
Alright sir